### PR TITLE
Deprecate kew, rewrite to be a slight wrapper on top of native Node Promises

### PR DESCRIPTION
**Pull request by mrblackus**

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
+    "outDir": "lib",
+    "module": "commonjs",
+    "target": "es2016",
+    "lib": [
+      "es2016",
+    ],
+    "moduleResolution": "node",
+	 "esModuleInterop": true,
```
Comment (wawrzy) [inline tsconfig.json:10]: np: extra space

File: tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    "strict": true,
+    "forceConsistentCasingInFileNames": true,
+    "allowSyntheticDefaultImports": true,
+    "noImplicitReturns": true,
+    "noImplicitThis": true,
+    "noImplicitAny": true,
+    "suppressImplicitAnyIndexErrors": true,
+    "noUnusedLocals": true,
+	 "skipLibCheck": true
```
Comment (wawrzy) [inline tsconfig.json:19]: same

**Review by wawrzy — APPROVED** [review]
🚀

**Review by AudreyBosdeveix — APPROVED** [review]

**Review by eponae — APPROVED** [review]
lgtm

**Review by zioyero — APPROVED** [review]
Looks good, might be worth staging a branch and letting our QA folks poke around a bit, since I expect there might be some things that probably depend on subtle bugs or implementation details of `Kew`. Considering this affects tutu, m2, and the whole jubilee stack it's probably worth doing our due diligence.

**Review by sidhu663 — APPROVED** [review]
+1 to staging / testing before using in m2 but should be fine to merge this now.

**Pull request merged**